recipe/meta.yaml

```diff
@@ -1,15 +1,17 @@
-{% set version = "3.17.2" %}
+{% set version = "3.19.1" %}
 
 package:
   name: protobuf
   version: {{ version }}
 
 source:
   url: https://github.com/protocolbuffers/protobuf/archive/v{{ version }}/protobuf-v{{ version }}.tar.gz
-  sha256: 36f81e03a0702f8f935fffd5a486dac1c0fc6d4bae1cd02c7a32448ad6e63bcb
+  sha256: 87407cd28e7a9c95d9f61a098a53cf031109d451a7763e7dd1253abf8b4df422
 
 build:
   number: 0
+  missing_dso_whitelist:  # [s390x]
+    - $RPATH/ld64.so.1    # [s390x]
 
```

Comment:
There should be `skip: True  # [py<35]`
see https://github.com/protocolbuffers/protobuf/blob/7c40b2df1fdf6f414c1c18c789715a9c948a0725/python/setup.py#L312